components/widgets/menu/menu_group.scss

```diff
@@ -6,6 +6,6 @@
     &.menu-divider {
         height: 1px;
         margin: 8px 0;
-        background: rgba(var(--center-channel-color-rgb), 0.16);
+        background: rgba(var(--center-channel-color-rgb), 0.08);
```

Comment:
Is this a permanent change whether GH is enabled or not? If it's only meant for when GH is enabled, you should probably move to `global_header.scss` until the feature flag is removed.

Reply:
I was actually wondering about this too. Since I worked on another ticket in the past which was about reducing the opacity on all dividers. @michaelgamble is this to be applied in all instances for menu dividers?